builders/interact/truffle.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Truffle使用教程
+description: 通过本教程，学习如何通过Truffle将基于Solidity的智能合约轻松部署到Moonbeam。
+---
+
+# 使用Truffle部署至Moonbeam
+
+<style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/RD5MefSPNeo' frameborder='0' allowfullscreen></iframe></div>
+<style>.caption { font-family: Open Sans, sans-serif; font-size: 0.9em; color: rgba(170, 170, 170, 1); font-style: italic; letter-spacing: 0px; position: relative;}</style>
+
+## 概览 {: #introduction }
+
+[Truffle](https://www.trufflesuite.com/)是一个以太坊智能合约集成开发工具。本教程介绍了如何通过Truffle将基于Solidity的智能合约部署到Moonbeam节点。鉴于Moonbeam兼容以太坊的特性，Truffle可与Moonbeam节点一起直接使用。
+
+!!! 注意事项
+    本教程用[Moonbase Alpha](https://github.com/PureStake/moonbeam/releases/tag/{{ networks.moonbase.version }}) {{ networks.moonbase.version }}版本的{{ networks.development.build_tag}}标签建立。为实现与以太坊的全面兼容，基于Substrate的Moonbeam平台和[Frontier](https://github.com/paritytech/frontier)组件正处于积极开发阶段。
+
+​    --8<-- 'text/common/assumes-mac-or-ubuntu-env.md'
```

Comment:
this text snippet isn't rendering on the page correctly

Reply:
fixed